tests/settings/test_settings_text_separators_meilisearch.py

```diff
@@ -0,0 +1,71 @@
+NEW_SEPARATOR_TOKENS = ["|", "&hellip;"]
+NEW_NON_SEPARATOR_TOKENS = ["@", "#"]
+
+def test_get_separator_tokens_default(empty_index):
```

Comment:
The reason black is failing is because there should be 2 spaces between functions (only 1 between class methods so that is why it didn't fail there). 

You can run `black meilisearch tests` and it will auto fix it for you.

Reply:
It's strange because I did run `black meilisearch tests` locally and it said there was one change. Oh well, I'll try again.